Riot/Modules/Common/KeyboardAvoiding/KeyboardAvoider.swift

```diff
@@ -54,6 +56,21 @@ final class KeyboardAvoider {
         self.unregisterKeyboardNotifications()
     }
     
+    /// Scroll to the given view, which must be a view in the scrollView.
+    /// - Parameters:
+    ///   - view: The view to scroll
+    ///   - animated: animate the scroll
+    func scroll(to view: UIView, animated: Bool = true) {
```

Comment:
This is good to have a dedicated method to encapsulate the scroll to a view in a scroll view. But this is not the responsibility of the `KeyboardAvoider`, which the goal is just to avoid the keyboard currently it only handles scrollviews but it can evolve.
Plus we have a magic number `-20` for  `rect = rect.insetBy(dx: 0, dy: -20)`. Either we define this method only in `AuthenticationViewController` or we can make a generic method we put as extension of `UIScrollView` like:
```
extension UIScrollView {
func vc_scroll(to view: UIView, with insets: CGRect, animated: Bool = true) {
```

Reply:
Sorry, i thought you wanted to have it in KeyboardAvoider. Updated.